torch/distributed/_tensor/debug/__init__.py

```diff
@@ -0,0 +1,10 @@
+from torch.distributed._tensor.api import DTensor
+
+
+def get_sharding_prop_cache_info():
```

Comment:
Thanks for adding this. It would be nice to add a comment or a reference on Python LRU so that user know what information we can get from this API. (This can be done in a follow-up PR as a BE work)

Reply:
Yeah sure I'll add comments in a follow up PR :)